docs/installation/index.rst

```diff
@@ -41,22 +38,19 @@ This section contains information on setting up and managing the Open Data Cube.
 
 
 .. toctree::
-
     :caption: Indexing Data
 
     indexing-data/step-guide
     indexing-data/indexing-from-s3
 
 
 .. toctree::
-
     :caption: Advanced Topics
 
     extending-odc
 
 
 .. toctree::
-
     :caption: Legacy Approaches
 
     data-preperation-scripts
```

Comment:
"preperation" should be "preparation".

Reply:
yes, this was picked up by spelling check, unless i change file name its not just one typo